### PR TITLE
Update presses-universitaires-de-rennes-archeologie-et-culture.csl

### PR DESCRIPTION
**Pull request by FrancoisC37**
Hello, 
With the first users review i've fix a "coma problem" in the citation

### CODE REVIEW

File: presses-universitaires-de-rennes-archeologie-et-culture.csl [hunk 1]
```diff
@@ -414,14 +414,10 @@
...
-        <group delimiter="&#160;">
-          <text macro="author-citation"/>
-          <text macro="year-date"/>
-        </group>
-        <text variable="locator"/>
-      </group>
-    </layout>
+     <text macro="author-citation" suffix=", "/>
+     <text macro="year-date" suffix=",&#160;p.&#160;"/>
+     <text variable="locator"/>
```
Comment (rmzelle) [inline presses-universitaires-de-rennes-archeologie-et-culture.csl]: @FrancoisC37, does the "p." stand for "page" here? If so, it doesn't belong on the "year-date" macro call. Instead, you could replace `<text variable="locator"/>` by:

```xml
<group delimiter="&#160;">
  <label variable="locator" form="short"/>
  <text variable="locator"/>
</group>
```

(and reduce the suffix on "year-date" to `suffix=",&#160;"`)

File: presses-universitaires-de-rennes-archeologie-et-culture.csl [hunk 2]
```diff
@@ -414,14 +414,13 @@
...
-        <text variable="locator"/>
-      </group>
-    </layout>
+     <text macro="author-citation" suffix=", "/>
+     <text macro="year-date" suffix=",&#160;"/>
+     <group delimiter="&#160;">
+       <label variable="locator" form="short"/>
+       <text variable="locator"/>
+     </group>
+   </layout>
```
Comment (rmzelle) [inline presses-universitaires-de-rennes-archeologie-et-culture.csl]: Thanks. This still doesn't look quite right, though. Could you try replacing:

```xml
     <text macro="author-citation" suffix=", "/>
     <text macro="year-date" suffix=",&#160;"/>
     <group delimiter="&#160;">
       <label variable="locator" form="short"/>
       <text variable="locator"/>
     </group>
```

by

```xml
     <group delimiter=",&#160;">
       <group delimiter=", ">
         <text macro="author-citation"/>
         <text macro="year-date"/>
       </group>
       <group delimiter="&#160;">
         <label variable="locator" form="short"/>
         <text variable="locator"/>
       </group>
     </group>
```
?

**Comment by rmzelle** [comment]
Great, thanks.

**Comment by FrancoisC37** [comment]
Thank you! It's a pleasure to do a style with your help!

2016-11-26 19:35 GMT+01:00 Rintze M. Zelle <notifications@github.com>:

**Pull request merged**